### DISCUSSION TITLE
Regex resources/Intermediate JS

### DISCUSSION THREAD

**Discussion by wcjameson** [wjwat/lhtlhtp.com#8] [General]
Hey I think am going to try and add some RegEx resources under the intermediate JS section.  Does that seem like a good spot for it?
Although I still don't entirely know what i'm doing :)

**Comment by wjwat** [comment]
The PR you submitted looks good to me (minus a few minor nits I had to pick). As for structure I can say I'm still a bit lost on what would be the best path forward so even if we end up having to rework things in the future I think there's value in just having something down on paper (so to speak).
